packages/bruno-app/src/components/Modal/StyledWrapper.js

```diff
@@ -1,6 +1,6 @@
 import styled from 'styled-components';
 
-const Wrapper = styled.div`
+const Wrapper = styled.dialog`
   color: ${(props) => props.theme.text};
```

Comment:
Could you also add:

```css
  max-height: unset;
  max-width: unset;
```
To remove the weird gap around the backdrop?

Reply:
could you please elaborate more

Reply:
Dialogs now have a "weird" gap between the window border and the backdrop:
![image](https://github.com/usebruno/bruno/assets/39559178/9a979cd1-6118-4d4e-b784-9dd37b9050af)

This is because the `dialog` Element has some weird default CSS. From the [chrome source](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/third_party/blink/renderer/core/html/resources/html.css#1444):

```css
dialog:-internal-dialog-in-top-layer {
    /* https://html.spec.whatwg.org/multipage/rendering.html#flow-content-3 */
    position: fixed;
    overflow: auto;
    inset-block-start: 0;
    inset-block-end: 0;
    max-width: calc(100% - 6px - 2em);
    max-height: calc(100% - 6px - 2em);
    /* https://github.com/w3c/csswg-drafts/issues/6939#issuecomment-1016679588 */
    user-select: text;
    visibility: visible;
}
```

Unsetting the default `max-width` and `max-height` will remove the gap.

```suggestion
  color: ${(props) => props.theme.text};
  
  max-height: unset;
  max-width: unset;
```

Reply:
thanks !!

fixed it now

Reply:
@Its-treason can you check now